xdsl/interactive/app.py

```diff
@@ -347,6 +389,49 @@ def get_query_string(self) -> str:
         )
         return f"xdsl-opt -p {query}"
 
+    def update_input_operation_count_tuple(self, input_text: str) -> None:
```

Comment:
can you change this to take a module instead, please?

Reply:
ok